releasenotes/notes/max-stream-duration.yaml

```
@@ -0,0 +1,7 @@
+apiVersion: release-notes/v2
+kind: bug-fix
+area: networking
+issue: []
```

If the issue number is not available, we can add the PR id here so the release notes generator will link PR in the release notes.

```suggestion
issue: 30885
```

---

Ok. thank you. Added